### PR TITLE
Code Rub: Fix One Liner Exam Fee Exceptions Remove By ID

### PR DESCRIPTION
**Pull request by devanteconnor**

### CODE REVIEW

File: OtripleS.Web.Api.Tests.Unit/Services/Foundations/AssignmentAttachments/AssignmentAttachmentServiceTests.cs [hunk 1]
```diff
@@ -25,8 +25,10 @@ public partial class AssignmentAttachmentServiceTests
 
         public AssignmentAttachmentServiceTests()
         {
-            this.storageBrokerMock = new Mock<IStorageBroker>();
-            this.loggingBrokerMock = new Mock<ILoggingBroker>();
+            this.storageBrokerMock =
+                new Mock<IStorageBroker>();
+            this.loggingBrokerMock =
+                new Mock<ILoggingBroker>();
```
Comment (ShriHumrudha) [inline OtripleS.Web.Api.Tests.Unit/Services/Foundations/AssignmentAttachments/AssignmentAttachmentServiceTests.cs:32]: This change is not required

File: OtripleS.Web.Api.Tests.Unit/Services/Foundations/AssignmentAttachments/AssignmentAttachmentServiceTests.cs [hunk 2]
```diff
@@ -61,7 +64,8 @@ public AssignmentAttachmentServiceTests()
 
         private static Filler<AssignmentAttachment> CreateAssignmentAttachmentFiller(DateTimeOffset dates)
         {
-            var filler = new Filler<AssignmentAttachment>();
+            var filler =
```
Comment (ShriHumrudha) [inline OtripleS.Web.Api.Tests.Unit/Services/Foundations/AssignmentAttachments/AssignmentAttachmentServiceTests.cs:67]: this is not required

**Review by ShriHumrudha — CHANGES_REQUESTED** [review]

**Comment by ShriHumrudha** [comment]
Fix your PR title too.

**Comment by hassanhabib** [comment]
@devanteconnor there are two things required here:
1. Revert the changes that @ShriHumrudha indicated are not needed on the same branch.
2. Make sure you properly name the PR.

**Comment by ShriHumrudha** [comment]
Closing the PR as it is beyond the 3-day window.

**Pull request closed without merging**